.github/workflows/integrationTests.yaml

```diff
@@ -168,6 +168,7 @@ jobs:
             verify
         env:
           DB_PASSWORD: ${{ secrets.SPRING_CLOUD_GCP_CI_DB_ROOT_PASSWORD }}
+          GOOGLE_CLOUD_PROJECT: spring-cloud-gcp-ci
```

Comment:
Why is the `GOOGLE_CLOUD_PROJECT` needed here, but not in the equivalent native env block? Both use `setup-gcloud` to set the `project_id`.

Reply:
Good question. On a second look, I updated this pr to use `GCP_PROJECT` which is commonly setup in both workflows.

IntegrationTest.yaml is using "google-github-actions/setup-gcloud@v0" vs. NativeTests.yaml using "google-github-actions/setup-gcloud@v1". That might have caused the slight difference in env vars setup.
[Integration](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/actions/runs/7029684999/job/19127758345)
```
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
    GCLOUD_PROJECT: spring-cloud-gcp-ci
```

[Native](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/actions/runs/7024465739/job/19152484632?pr=2384):
```
    CLOUDSDK_CORE_PROJECT: spring-cloud-gcp-ci-native
    CLOUDSDK_PROJECT: spring-cloud-gcp-ci-native
    GCLOUD_PROJECT: spring-cloud-gcp-ci-native
    GCP_PROJECT: spring-cloud-gcp-ci-native
    GOOGLE_CLOUD_PROJECT: spring-cloud-gcp-ci-native
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
    CLOUDSDK_METRICS_ENVIRONMENT_VERSION: 1.1.1
```
I'll raise a separate pr to upgrade this github action version for integration yaml.